drivers/Ticker.h

```diff
@@ -115,9 +115,11 @@ class Ticker : public TimerEvent, private NonCopyable<Ticker> {
     void attach_us(Callback<void()> func, us_timestamp_t t) {
         core_util_critical_section_enter();
         // lock only for the initial callback setup and this is not low power ticker
+#if DEVICE_SLEEP
```

Comment:
this fix should be a separate PR ! We provide empty implementation fro sleep manager so it would end up being removed anyway (does not need to have sleep protect in the user code).